projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -793,6 +793,12 @@ protected function render_option_keys( &$options_response_keys ) {
 						$options[ $key ] = $site_creation_flow;
 					}
 					break;
+				case 'site_source_slug':
```

Comment:
We need to add an entry to the `$site_options_format` array to expose it through API.